If-risk/Exceptions/PolicyAlreadyExistsException.cs

```diff
@@ -0,0 +1,22 @@
+namespace If_risk.Exceptions;
+
+[Serializable]
```

Comment:
kam šis bija vajadzīgs?

Reply:
Kad skatījos, kā veidot custom exceptions gandrīz visur to izmantoja
Palasīju par to un izskatās, ka tiešām nav vajadzīgs